packages/dds/map/src/interfaces.ts

```diff
@@ -344,20 +345,21 @@ export interface ISharedMapEvents extends ISharedObjectEvents {
 
 /**
  * Shared map interface
+ * TODO: what are the semantics of this?
  */
 export interface ISharedMap extends ISharedObject<ISharedMapEvents>, Map<string, any> {
```

Comment:
The variation of this description from `SharedMap` may be appropriate here:

It provides the same API for setting and retrieving values that JavaScript developers are accustomed to with the {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map | Map} built-in object.